kube-spawn: add start/stop commands

dongsupark opened the pull request:
* rename command from `up` to `setup`
* split out commands into multiple files to maintain the source tree better. `init.go` for init command, `setup.go` for setup command etc.
* introduce `start` and `stop` commands. `start` wraps `machinectl pull ... coreos`, `setup` and `init`. `stop` does what `machinectl stop kube-spawn-{0,...,n}` would achieve

Fixes https://github.com/kinvolk/kube-spawn/issues/54

robertgzr commented:
Also closes #73

robertgzr commented:
I see that the `stop` command also removes the images. Can't we have that as two separate commands? Like `stop` to just bring down the cluster and `destroy` to remove the images? ( #38 )

I would expect that if I run:
```
$ kube-spawn start
$ kubectl ...
$ kube-spawn stop
```
That I can just get my cluster back up by just running `kube-spawn start` again.
I guess that still needs #37 right?

dongsupark commented:
@robertgzr Thinking again about it closely, I think you're right. It makes sense to split it out into 2 separate commands: `kube-spawn stop` for `machinectl poweroff` (and/or `terminate`), and `kube-spawn destroy` for `machinectl remove`.
I'm going to exclude the latter from this PR, to create another PR to address #38.

As far as I see, without removing images, it would sometimes fail to restart a machine. To fix it, #37 needs to be implemented in the future. Users probably need to keep in mind the restriction.

dongsupark commented:
Updated.

* Excluded parts for removing images.
* Added an option `'--force'` to the `stop` command to run `machinectl terminate` instead of `poweroff`.
* Updated `scripts/vagrant-setup-env.sh` to replace `up` with `setup`.
* Let the `up` command to print out only a deprecation warning.

dongsupark commented:
Also updated README.
